src/expressions/functions/builtInFunctions.node.ts

```diff
@@ -112,6 +112,84 @@ const fnHasChildren: FunctionDefinitionType = function(
 	});
 };
 
+const fnPath: FunctionDefinitionType = function(
```

Comment:
Could you also change this function to an arrow expression as that is preferred in our tslint configuration.

Reply:
it is done